README.rst

```diff
@@ -35,12 +35,12 @@ GitHub repository:
 
     $ git clone https://github.com/cyclus/cymetric
 
-Then build and install:
+Then build and install to the same location Cyclus is installed:
```

Comment:
We can, but is there any evidence to support that? My understanding is that it should just work (tm). If anything would cause problems, it is my understanding that the cyclus bindings may be installed to a location that may not be on the default PYTHONPATH on Mac for the system Python. (Though I think ~/.local is always on the PYTHONPATH). So if anything the warning should be in cyclus, not cymetric.